### PR TITLE
Add error handling in generateClient

### PR DESCRIPTION
**Pull request by danamansana**

### CODE REVIEW

**Comment by ktp242** [comment]
Also the documentations with Changlog and readme and package.json
Other than that I think it looks good.

**Comment by danamansana** [comment]
Updated changelog, readme, and package.json

**Comment by ktp242** [comment]
Looks good! Since I will be out tomorrow, I think you can deploy to Dev server and test it out. If no issues, I think we can merge it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## Changelog
 
 ### v1.1.1
+#### Added
+- Added error handling for adding caching
```
Comment (ktp242) [inline CHANGELOG.md]: Btw, try to use the same verb tense? It is "Added" here while line 7 is "Adding".

**Comment by ktp242** [comment]
You can try disable the env variables or put a wrong one. This way I think the app should get an error response.

**Pull request merged**